app.js

```diff
@@ -15,5 +15,50 @@ app.get('/', (req, res) => {
   res.send('oh hello')
 });
 
+app.get('/api/v1/projects/', async(request, response) => {
+  try {
+    const project = await database('projects').select();
```

Comment:
Naming convention might be better to have this variable called `projects` rather than `project`, to signify this is more than one value. Just a thought

Reply:
Good catch, I'll update it and get it pushed up!